styles/pages/PageOffine.module.sass

```
@@ -1,3 +1,3 @@
 @use '@/styles/utils/mixins' as m
 @use '@/styles/utils/funcs' as f
-@use '@/styles/themes/default/Colors.module.sass' as c
\ No newline at end of file
+@use '@/styles/themes/def/Colors.module.sass' as c
```

should be default because it's easier to read. `def` for `default` is not as common
```suggestion
@use '@/styles/themes/default/Colors.module.sass' as c
```